backend/src/Edna/Web/API.hs

```diff
@@ -40,7 +40,7 @@ data EdnaEndpoints route = EdnaEndpoints
 type EdnaAPI =
   ToServant EdnaEndpoints AsApi
   :<|>
-  "health" :> Summary "Check the health of this server" :> GetNoContent
+  "metrics" :> "health" :> Summary "Check the health of this server" :> GetNoContent
```

Comment:
I already commented in Slack, but I’ll repeat here: I find this renaming pretty weird.

Reply:
@mkaito I guess you can answer here. Does we really need `health` endpoint for now?

Reply:
In a way, he's right. We can remove it. We can just react to the fact that the Prometheus endpoint responds or not.

Reply:
done